meteor/client/lib/rundownTiming.ts

```diff
@@ -523,6 +547,17 @@ export class RundownTimingCalculator {
 					// offset the parts before the on air line by the countdown for the end of the rundown
 					this.linearParts[i][1] =
 						(this.linearParts[i][1] || 0) + waitAccumulator - localAccum + currentRemaining
+					if (nextBreakStartTime === undefined) {
+						const backTime = getBreakStartTime(partInstancesMap, segments, this.linearParts[i][0])
+						if (backTime) {
+							nextBreakStartTime = backTime
+						}
+
+						const cumeTime = getBreakEndTime(partInstancesMap, segments, this.linearParts[i][0])
+						if (cumeTime) {
+							nextBreakEndTime = cumeTime
+						}
+					}
```

Comment:
Is this top-level `if` block correct? Shouldn't it break at any of these two being set, as opposed to just one?